public/index.ts

```diff
@@ -880,10 +880,7 @@ function loadImageData(jsonData: ImageInfo, volumeData: Uint8Array[]) {
 function onChannelDataArrived(v: Volume, channelIndex: number) {
   const currentVol = v; // myState.volume;
 
-  const hmin = currentVol.getHistogram(channelIndex).findBinOfPercentile(0.5);
-  const hmax = currentVol.getHistogram(channelIndex).findBinOfPercentile(0.983);
-  const lut = new Lut().createFromMinMax(hmin, hmax);
-  currentVol.setLut(channelIndex, lut.lut);
```

Comment:
removing this code allows the automatic Lut remapping to happen in the test app.  (Happens before this callback is fired)